pytest/tests/loadtest/locust/common/base.py

```diff
@@ -277,16 +277,24 @@ def send_tx(self, tx: Transaction, locust_name: str) -> dict:
             try:
                 # To get proper errors on invalid transaction, we need to use sync api first
```

Comment:
```suggestion
                # To get proper errors on invalid transaction, we need to wait at least for optimistic execution
```
maybe that's more precise?

Reply:
Thanks, accepted!